### PR TITLE
Swap details accordion

### PR DESCRIPTION
**Pull request by iceanddust**

### CODE REVIEW

File: packages/widgets/src/SwapWidget/SwapDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <Text>
+          <AccordionButton sx={{ px: 0, _hover: { bg: 'none' } }}>
+            <Box flex="1" textAlign="left">
+              <Trans>Swap details</Trans>
+            </Box>
+            <AccordionIcon />
+          </AccordionButton>
+        </Text>
+        <AccordionPanel px={1}>
+          {chainId === 1 && quoteData.estimatedGasUsedUSD && (
```
Comment (codewiththeflowxyz) [inline packages/widgets/src/SwapWidget/SwapDetails.tsx:38]: Why does this only apply to mainnet?

**Review by codewiththeflowxyz — APPROVED** [review]
LGTM, just one question.

File: packages/widgets/src/SwapWidget/SwapDetails.tsx (hunk 1 above)
Comment (iceanddust) [reply packages/widgets/src/SwapWidget/SwapDetails.tsx]: I added the estimation in ETH to Goerli as the estimation in USD is not applicable

**Review by generatorpoint — APPROVED** [review]
LGTM

**Review by bitsforpeace — APPROVED** [review]

**Pull request merged**